src/test/vscodeUiTest/ProjectGenerationWizard.ts

```diff
@@ -140,8 +140,9 @@ export class ProjectGenerationWizard extends InputBox {
    */
   public async getBackButton(): Promise<WebElement | undefined> {
     const enclosing: WebElement = this.getEnclosingElement();
+    const backButtonClass: string = 'codicon-quick-input-back';
     try {
-      const backButton: WebElement = await enclosing.findElement(By.className('quick-input-button-icon-1'));
+      const backButton: WebElement = await enclosing.findElement(By.className(backButtonClass));
```

Comment:
The back button currently is not supported by vscode-extension-tester: https://github.com/redhat-developer/vscode-extension-tester/issues/136, therefore I hardcoded the back button's class name.